package.json

```diff
@@ -58,25 +61,25 @@
     }
   },
   "scripts": {
-    "prepare": "husky install && (forge --version || yarn foundryup) && yarn workspaces run prepare",
+    "prepare": "husky install && (forge --version || foundryup) && pnpm recursive run prepare",
     "commit": "cz",
     "prettier:check": "prettier --check 'packages/**/*.ts'",
     "prettier": "prettier --write 'packages/**/*.ts'",
     "lint": "eslint . --ext .ts",
-    "release": "yarn && lerna publish --no-private --force-publish && yarn retype:updateversion",
+    "release": "pnpm install && lerna publish --no-private --force-publish && pnpm retype:updateversion",
     "release:manual": "node scripts/workspaceRun.mjs release",
-    "release:yalc": "(yarn entry:dist && yarn lerna exec npx yalc push); yarn entry:src",
+    "release:yalc": "(pnpm entry:dist && lerna exec npx yalc push); pnpm entry:src",
     "foundryup": "curl -L https://foundry.paradigm.xyz | bash && bash ~/.foundry/bin/foundryup",
-    "link:packages": "yarn lerna run link",
-    "docs": "yarn docs:prepare && yarn retype build",
-    "docs:prepare": "yarn lerna run docs && yarn retype:updateversion",
-    "docs:dev": "yarn retype watch .",
-    "retype:updateversion": "sed \"s/label: .*/label: $(yarn list --pattern @latticexyz/solecs | grep -e @latticexyz | sed \"s/.*@//\")/\" retype.yml > retype.yml.tmp && mv retype.yml.tmp retype.yml",
-    "test": "yarn workspaces run test",
-    "entry:dist": "yarn lerna run prepack",
-    "entry:src": "yarn lerna run postpack",
-    "gas-report": "yarn workspace @latticexyz/store run gas-report && yarn workspace @latticexyz/world run gas-report",
-    "tablegen": "yarn workspace @latticexyz/store run tablegen && yarn workspace @latticexyz/world run tablegen"
+    "link:packages": "lerna run link",
+    "docs": "pnpm docs:prepare && retype build",
+    "docs:prepare": "lerna run docs && pnpm retype:updateversion",
+    "docs:dev": "retype watch .",
+    "retype:updateversion": "sed \"s/label: .*/label: $(pnpm list --filter @latticexyz/solecs | grep -e @latticexyz | sed \"s/.*@//\")/\" retype.yml > retype.yml.tmp && mv retype.yml.tmp retype.yml",
+    "test": "pnpm recursive run test",
+    "entry:dist": "lerna run prepack",
+    "entry:src": "lerna run postpack",
+    "gas-report": "workspace @latticexyz/store run gas-report && pnpm recursive run --filter @latticexyz/world run gas-report",
+    "tablegen": "pnpm recursive run --filter @latticexyz/store run tablegen && pnpm recursive run --filter @latticexyz/world run tablegen"
```

Comment:
since we're on pnpm now, I think this can just be `pnpm --parallel gas-report` and `pnpm --parallel tablegen` (and pnpm is smart enough to skip packages where the script is not defined)

Reply:
Ah I see yea yarn workspaces are annoying about that in yarn classic